Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -1062,6 +1076,9 @@ struct WebCore::ApplePayLineItem {
 #if ENABLE(APPLE_PAY_AUTOMATIC_RELOAD_LINE_ITEM)
     String automaticReloadPaymentThresholdAmount
 #endif
+#if ENABLE(APPLE_PAY_DISBURSEMENTS)
+    std::optional<WebCore::ApplePayLineItem::DisbursementLineItemType> disbursementLineItemType
+#endifclean
```

Comment:
This might be messing with the serializer generation
```suggestion
#endif
```

Reply:
Cool let's hope this works